types/danish-ssn/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for danish-ssn 2.1.1
+// Project: https://github.com/mathiasvr/danish-ssn/
+// Definitions by: Yuriy Guzenko <https://github.com/yuriyg86>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace cpr {
+    export function isValid(ssn: string): boolean;
+    export function validate(ssn: string): string;
+    export function validForDate(date: Date): string[];
```

Comment:
```suggestion
    function isValid(ssn: string): boolean;
    function validate(ssn: string): string;
    function validForDate(date: Date): string[];
```
those are exported by default, `export` is not required here.